### PR TITLE
Updated UI for Mealplan View

### PR DESCRIPTION
**Pull request by nkompella**
Used bootstrap and flexbox.

### CODE REVIEW

**Review by cheolhojang — COMMENTED** [review]
is the recipe link still in the show page?

File: app/views/users/show.html.haml [hunk 1]
```diff
@@ -1,34 +0,0 @@
-%body
-  = render 'navbar'
-  %h2.text-center Recommended Daily Meal Plan
-  .wrapper
-    %header.header Your calories are #{@calories}
-    %aside.aside.aside-l
-      %h3 Breakfast: 
-      =link_to(@breakTitle, @breakLink)
-      
-      %h5 Calories: #{@breakCals}
```
Comment (omidafshar) [inline app/views/users/show.html.haml:10]: Why was this file deleted?

Comment (nkompella) [reply app/views/users/show.html.haml:10]: I converted the haml files to erb in order to make bootstrap integration easier!
